PermissionsEx/permissions.yml

```diff
@@ -998,6 +998,41 @@ groups:
         - ch.alias.haste3
         - ch.alias.carryplayer
         - ch.alias.dismountplayer
+      maui:
+    options:
+      prefix: '&3M&fa&6u&9i'
+      suffix: '&3'
+    inheritance:
+    - mortal
+    worlds:
+      survival:
+        permissions:
+        - essentials.fly
+        - essentials.speed.fly
```

Comment:
He Can Turn Into An Eagle Cause He Is A ShapeShifter. I Dont Need To Give A website it's on the commercial